collector/README.md

```diff
@@ -88,4 +88,70 @@ from an S3 object using a CloudFormation template:
           OPENTELEMETRY_COLLECTOR_CONFIG_FILE: s3://<bucket_name>.s3.<region>.amazonaws.com/collector_config.yaml
 ```
 
-Loading configuration from S3 will require that the IAM role attached to your function includes read access to the relevant bucket.
\ No newline at end of file
+Loading configuration from S3 will require that the IAM role attached to your function includes read access to the relevant bucket.
+
+# Improving Lambda responses times
+At the end of a lambda function's execution, the OpenTelemetry client libraries will flush any pending spans/metrics/logs
+to the collector before returning control to the Lambda environment. The collector's pipelines are synchronous and this 
+means that the response of the lambda function is delayed until the data has been exported. 
+This delay can potentially be for hundreds of milliseconds.
+
+To overcome this problem the [decouple](./processor/decoupleprocessor/README.md) processor can be used to separate the 
+2 ends of the collectors pipeline, allow the lambda function to complete and still ensuring that any data is exported 
+before the Lambda environment is frozen.
+
+Below is a sample configuration that uses the decouple processor:
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  logging:
+    loglevel: debug
+  otlp:
+    endpoint: { backend endpoint }
+
+processors:
+  decouple:
+
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [decouple]
+      exporters: [logging, otlp]
+```
+
+## Reducing Lambda runtime
+If your lambda function is invoked frequently it is also possible to pair the decouple processor with the batch 
+processor to reduce total lambda execution time at the expense of the delaying the export of OpenTelemetry data. 
+When used with the batch processor the decouple processor must be the last processor in the pipeline to ensure that data
```

Comment:
```suggestion
When using both the decouple and batch processors, the decouple processor must be last in the pipeline to ensure that data
```
Just offering the suggestion above to try and say this with one less instance of "processor."